src/constants.js

```diff
@@ -21,14 +21,38 @@ export const SESSION_TYPES = {
   CUSTOMER: "CUSTOMER"
 };
 
+export const CSM_CATEGORY = {
+  API: "API",
+  UI: "UI"
+};
+
+export const ACPS_METHODS = {
+  SEND_MESSAGE: "SendMessage",
+  SEND_ATTACHMENT: "SendAttachment",
+  DOWNLOAD_ATTACHMENT: "DownloadAttachment",
+  SEND_EVENT: "SendEvent",
+  GET_TRANSCRIPT: "GetTranscript",
+  DISCONNECT_PARTICIPANT: "DisconnectParticipant",
+  CREATE_PARTICIPANT_CONNECTION: "CreateParticipantConnection",
+};
+
+export const WEBSOCKET_EVENTS = {
+  ConnectionLost: "WebsocketConnectionLost",
+  ConnectionGained: "WebsocketConnectionGained",
+  Ended: "WebsocketEnded",
+  IncomingMessage: "WebsocketIncomingMessage",
+  InitWebsocket: "InitWebsocket"
+};
+
 export const CHAT_EVENTS = {
   INCOMING_MESSAGE: "INCOMING_MESSAGE",
   INCOMING_TYPING: "INCOMING_TYPING",
   CONNECTION_ESTABLISHED: "CONNECTION_ESTABLISHED",
   CONNECTION_LOST: "CONNECTION_LOST",
   CONNECTION_BROKEN: "CONNECTION_BROKEN",
   CONNECTION_ACK: "CONNECTION_ACK",
-  CHAT_ENDED: "CHAT_ENDED"
+  CHAT_ENDED: "CHAT_ENDED",
+  START_CHAT_SESSION: "StartChatSession"
```

Comment:
Can you make it consistent on styling - "START_CHAT_SESSION"

Reply:
yeah, I considered that before, but other csm methods(like acps methods) are using camel case, so I follow that style.

Reply:
In that case can you move this out of CHAT_EVENTS because it's not an event like others? All the others are real events from SendEvent.  You can create a separate constant for this